Gemfile

```diff
@@ -9,7 +9,7 @@ gem "logging", "2.1.0"
 gem 'nokogiri', "1.6.7.2"
 gem 'whenever', require: false
 gem "slop", "3.4.5"
-gem "sidekiq", "< 3"
+gem "sidekiq", "< 4"
```

Comment:
shouldn't we lock it to `3.4.1` instead? what's the chances of someone upgrading to 3.5.4 ?